src/types/memory-controller.ts

```diff
@@ -0,0 +1,34 @@
+import { CardData, GameData } from "./data-models";
+import { testData } from "./test-data";
+
+export class MemoryController {
+  gameData: GameData;
+  constructor() {
+    const gameDataDto: GameDataDTO = this.fetchData();
+    this.gameData = this.convertDTOToData(gameDataDto);
+    //this.shuffleCards();
```

Comment:
why shuffle disabled?

Reply:
just for testing purposes, can be enabled later